src/view/signUpView.java

```diff
@@ -48,6 +51,8 @@ private void placeComponents(JPanel panel){
         JButton signUp = new JButton(signUpViewModel.SIGNUP_BUTTON_LABEL);
         buttons.add(signUp);
```

Comment:
- Good that you have a button so user can go to sign up view
- you will probably do this later but needs an action method